doc/nrf/glossary.rst

```diff
@@ -126,6 +132,11 @@ Glossary
    Data Terminal Ready (DTR)
       A control signal in RS-232 serial communications transmitted from data terminal equipment, such as a computer, to data communication equipment.
 
+   Delivery Traffic Indication Message (DTIM)
```

Comment:
Worth adding TIM as well

Reply:
@udaynordic please review this. I don't think this definition is correct - it doesn't follow the beacon, but rather is an element of the beacon ? And not sure we need unicast/multicast/broadcast discussion.

Reply:
DTIM is a message in a beacon sent for every DTIM period. This is a message present in the beacon frame. 
from spec:  11.2.1.3 TIM types
Two different TIM types are distinguished: TIM and DTIM. After a DTIM, the AP shall send out the buffered
broadcast/multicast MSDUs using normal frame transmission rules, before transmitting any unicast
frames.
The AP shall transmit a TIM with every beacon. Every DTIMPeriod, a TIM of type “DTIM” is transmitted
within a beacon, rather than an ordinary TIM.

Reply:
@udaynordic Please, check whether the description is good enough now. Left out the unicast/multicast part and changed the beginning as in your suggestion.